### PR TITLE
EVAKA-4203 Initial child's Vasu and Leops view

### PR DESCRIPTION
**Pull request by kimifoud**
#### Summary

Implement initial child's Vasu and Leops view for dev purposes

### CODE REVIEW

File: frontend/src/employee-frontend/api/vasu.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Failure, Result, Success } from '../../lib-common/api'
+import { JsonOf } from '../../lib-common/json'
+import { UUID } from '../types'
+import {
+  deserializeVasuDocumentSummary,
+  VasuDocumentSummary
+} from '../types/vasu'
+import { client } from './client'
+
+export async function getVasuDocumentSummary(
```
Comment (Joosakur) [inline frontend/src/employee-frontend/api/vasu.ts:14]: rather getVasuDocumentSummaries?

File: frontend/src/employee-frontend/types/vasu.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import { JsonOf } from '../../lib-common/json'
+import { UUID } from './index'
+
+export enum VasuDocumentState {
+  DRAFT = 'DRAFT',
+  CREATED = 'CREATED',
+  REVIEWED = 'REVIEWED',
+  CLOSED = 'CLOSED'
+}
```
Comment (Joosakur) [inline frontend/src/employee-frontend/types/vasu.ts:13]: hmm haven't seen this before, no idea how it compares to `type VasuDocumentState = 'DRAFT' | 'CREATED' | ...`

File: frontend/src/employee-frontend/components/child-information/VasuAndLeops.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  const { vasus, setVasus } = useContext(ChildContext)
+
+  const [open, setOpen] = useState(startOpen)
+
+  useEffect(() => {
+    setVasus(Loading.of())
+    void getVasuDocumentSummary(id).then((response) => {
+      setVasus(response)
+    })
+  }, [id, setVasus])
```
Comment (Joosakur) [inline frontend/src/employee-frontend/components/child-information/VasuAndLeops.tsx:71]: might be better as
```
const loadVasus = useRestApi(getVasuDocumentSummary, setVasus)
useEffect(() => loadVasus(id), [id, loadVasus])
```

File: frontend/src/employee-frontend/components/child-information/VasuAndLeops.tsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        />
+        <Table>
+          <Thead>
+            <Tr>
+              <Th />
+              <Th />
+              <Th />
+              <Th />
+            </Tr>
+          </Thead>
```
Comment (Joosakur) [inline frontend/src/employee-frontend/components/child-information/VasuAndLeops.tsx:129]: should the whole thead be omitted if there's nothing in it anyway?

File: frontend/src/employee-frontend/components/ChildInformation.tsx [hunk 5]
```diff
@@ -95,6 +97,7 @@ const layouts: Layouts<typeof components> = {
     { component: 'backup-care', open: false },
     { component: 'service-need', open: false },
     { component: 'daily-service-times', open: false },
+    { component: 'vasuAndLeops', open: false },
```
Comment (Joosakur) [inline frontend/src/employee-frontend/components/ChildInformation.tsx:100]: even though this is just for admin, maybe we should introduce a new feature flag into config.ts and set it to false in prod and maybe staging too?

Comment (Joosakur) [reply frontend/src/employee-frontend/components/ChildInformation.tsx:100]: PS: Panu has feature flag in https://github.com/espoon-voltti/evaka/pull/1013 so sync with that

File: frontend/src/employee-frontend/components/child-information/VasuAndLeops.tsx (hunk 3 above)
Comment (kimifoud) [reply frontend/src/employee-frontend/components/child-information/VasuAndLeops.tsx:71]: Aye, absolutely. This was a bad copy-paste 😬

File: frontend/src/employee-frontend/types/vasu.ts (hunk 2 above)
Comment (kimifoud) [reply frontend/src/employee-frontend/types/vasu.ts]: Enum ends up in the generated js whereas `type` is just type info. Switched to type.

File: frontend/src/employee-frontend/components/ChildInformation.tsx (hunk 5 above)
Comment (kimifoud) [reply frontend/src/employee-frontend/components/ChildInformation.tsx]: Feature flag implemented.

**Review by Joosakur — APPROVED** [review]

**Pull request merged**